ui/seek_bar.js

```diff
@@ -134,6 +135,7 @@ shaka.ui.SeekBar = class extends shaka.ui.RangeElement {
    * @override
    */
   onChangeStart() {
+    this.wasPlaying_ = ! this.video.paused;
```

Comment:
style nit: please remove the space after the `!`

Reply:
done